assets/src/blocks/Timeline/TimelineFrontend.js

```diff
@@ -1,25 +1,24 @@
-import { Fragment } from '@wordpress/element';
 import { Timeline } from './Timeline';
```

Comment:
Not crucial for this block, since the resulting size is only 5.6KB (2.3KB compressed), but we could check whether there are things that are imported in these chunks that are actually already loaded by the main index file. Some things like `useEffect` are resolved to globals anyway, but others might be included in each entrypoint if they are imported?

Reply:
I took a look at the compiled code, I think wp.element is registered as external and so not included in this script (there are a bunch of dependencies noted as external) - not sure how it works exactly, but we can take a look at the other scripts when we split them

Reply:
Yeah should be fine for this one :+1: